attesters/csv/collect_evidence.c

```diff
@@ -81,6 +82,19 @@ static int do_hypercall(unsigned int p1, unsigned long p2, unsigned long p3)
 	return (int)ret;
 }
 
+static size_t curl_writefunc_callback(void *contents, size_t size, size_t nmemb, void *userp)
+{
+	size_t realsize = size * nmemb;
+
+	if (realsize != HYGON_HSK_CEK_CERT_SIZE) {
+		RATS_ERR("hsk_cek size is not equal to %d bytes.", HYGON_HSK_CEK_CERT_SIZE);
+		return 0;
+	}
+	memcpy(userp, contents, realsize);
+
+	return realsize;
+}
+
```

Comment:
`curl_writefunc_callback` may called times, please refer to the usage of this in `sev-snp` to modify it.